app/views/candidate_interface/personal_details/immigration_right_to_work/_shared_form.html.erb

```diff
@@ -1,12 +1,23 @@
 <%= f.govuk_error_summary %>
 
+<h1 class="govuk-heading-l">
+  <%= t('page_titles.right_to_work') %>
+</h1>
+
+<p class="govuk-body">
+  You’ll need the right to work or study in the UK for the duration of your course.
+</p>
+
+<p class="govuk-body">
+  <%= govuk_link_to 'Check if you\'ve already got the right to work or study in the UK, or whether you’ll need to apply for a visa', t('train_to_teach_in_england.url') %>.
```

Comment:
could we link to the relevant section of the guidance instead? here:

https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#visa